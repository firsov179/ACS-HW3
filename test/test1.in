1.570796326794896