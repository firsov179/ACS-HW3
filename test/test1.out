sin(1.570796) ~= 1.000000.
